### PR TITLE
Website accessibility increased

### PR DESCRIPTION
**Pull request by wojciech-kumon**

### CODE REVIEW

File: src/components/proposalFinder/FinderForm.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const ProposalIdLabel = styled(Label)`
 `;
 
 const ProposalIdInput = ({input, meta}) => (
-  <Input {...input} disabled={meta.submitting} bsSize="lg"/>
+  <Input {...input} disabled={meta.submitting} aria-label="Proposal ID" bsSize="lg"/>
```
Comment (patrykbuzowiczsabre) [inline src/components/proposalFinder/FinderForm.tsx]: not an expert, but isn't this redundant? what about `aria-labelledby` pointing at the `ProposalIdLabel`?

File: src/components/searchFlights/SearchFlightsForm.tsx [hunk 2]
```diff
@@ -21,6 +21,7 @@ export class SearchFlightsForm extends React.PureComponent<SearchFlightsFormProp
                 size="lg"
                 className="float-right mt-3"
                 disabled={submitting || invalid}
+                aria-label="Submit"
```
Comment (patrykbuzowiczsabre) [inline src/components/searchFlights/SearchFlightsForm.tsx]: is this helpful if the button has a descriptive content?

File: src/components/proposalFinder/FinderForm.tsx (hunk 1 above)
Comment (wojciech-kumon) [reply src/components/proposalFinder/FinderForm.tsx]: You're right. Only id in input is needed to enable `for="proposalId"`

File: src/components/searchFlights/SearchFlightsForm.tsx (hunk 2 above)
Comment (wojciech-kumon) [reply src/components/searchFlights/SearchFlightsForm.tsx]: removed

**Pull request merged**